playwright/tests/common/sideNav.spec.ts

```diff
@@ -60,6 +60,28 @@ test("I can navigate through all menu entries", async ({
   );
 });
 
+test("I can access the repository using the open icon", async ({
+  sliceMachinePage,
+  procedures,
+}) => {
+  procedures.mock("getState", ({ data }) => {
+    const result = data as { env: { manifest: { apiEndpoint: string } } };
+    result.env.manifest.apiEndpoint =
+      "https://example-prismic-repo.cdn.prismic.io/api/v2";
+    return result;
+  });
+
+  await sliceMachinePage.gotoDefaultPage();
+  await expect(sliceMachinePage.menu.repositoryLink).toBeVisible();
+
+  await sliceMachinePage.menu.repositoryLink.click();
+
+  const newTab = await sliceMachinePage.page.waitForEvent("popup");
```

Comment:
I made an error on my code like that that Baptiste tried to show me, but that I didn't see.

This is working, but not totally like the doc is doing: https://playwright.dev/docs/pages#handling-popups

As you can see they tell you to create the Promise without await before the click that trigger the popup event. I think it's better if we do like the doc wants us to do.

I fixed it already in my opened PR for the Academy test.

<img width="1017" alt="Screenshot 2024-03-08 at 10 17 58" src="https://github.com/prismicio/slice-machine/assets/19946868/d39629cd-ef57-40e0-847a-660f332d302e">

Reply:
Ah, interesting! Fixed here: https://github.com/prismicio/slice-machine/pull/1316/commits/ae0a66b222fe809ab2cfaaac89715841064b1bec